README.md

```diff
@@ -170,7 +170,7 @@ surrogate_keys:
 | `cdn`                                             | The CDN management, if you use any cdn to proxy your requests Souin will handle that                                                        |                                                                                                                           |
 | `cdn.provider`                                    | The provider placed before Souin                                                                                                            | `akamai`<br/><br/>`fastly`<br/><br/>`souin`                                                                               |
 | `cdn.api_key`                                     | The api key used to access to the provider                                                                                                  | `XXXX`                                                                                                                    |
-| `cdn.dynamic`                                     | Enable the dynamic keys returned by your backend application                                                                                | `true`<br/><br/>`(default: false)`                                                                                        |
+| `cdn.dynamic`                                     | Enable the dynamic keys returned by your backend application                                                                                | `true`<br/><br/>`(default: false)`<br/><br/>`(default: true)` in the Caddy plugin if `cdn` is provided                    |
```

Comment:
Can you update the `https://github.com/darkweak/souin/blob/master/plugins/caddy/README.md` too?

Reply:
Done !